app/src/renderer/vuex/modules/send.js

```diff
@@ -77,7 +77,8 @@ export default ({ node }) => {
         account_number: rootState.wallet.accountNumber,
         chain_id: rootState.connection.lastHeader.chain_id,
         gas: String(state.externals.config.default_gas),
-        generate_only: true
+        generate_only: true,
+        memo: `Sent via Cosmos UI 🚀`
```

Comment:
```suggestion
        memo: `Sent via Cosmos Voyager 🚀`
```

Reply:
Well it is not Voyager anymore in the future

Reply:
it's still Voyager, until we change the name

Reply:
(we're not `Cosmos UI` either)